cmd/adduser.go

```diff
@@ -311,3 +266,108 @@ func (p *AddUserParams) editUserClaim(c interface{}, ctx ActionCtx) error {
 
 	return nil
 }
+
+type ResponsePermsParams struct {
+	respTTL string
+	respMax string
+	rmResp  bool
+}
+
+func (p *ResponsePermsParams) maxResponseValidator(s string) error {
+	_, err := p.parseMaxResponse(s)
+	return err
+}
+
+func (p *ResponsePermsParams) parseMaxResponse(s string) (int, error) {
+	if s == "" {
+		return 0, nil
+	}
+	v, err := strconv.Atoi(s)
+	if err != nil {
+		return 0, err
+	}
+	return v, nil
+}
+
+func (p *ResponsePermsParams) ttlValidator(s string) error {
+	_, err := p.parseTTL(s)
+	return err
+}
+
+func (p *ResponsePermsParams) parseTTL(s string) (time.Duration, error) {
+	if s == "" {
+		return time.Duration(0), nil
+	}
+	d, err := time.ParseDuration(s)
```

Comment:
Probably same, likely `return time.ParseDuration(s)` here would do the same (did not check what ParseDuration returns as duration in case of error, but again, likely caller is not going to use the duration if error returned).

Reply:
if validation fails, the whole thing stops at that point - unless you are interactively editing. in which case you continue to try. After that, the value is never going to be an error as it already passed, but I need to handle it or it is not obvious. IE - if there's an error it won't ever record it